saq/queue/postgres.py

```diff
@@ -439,7 +439,6 @@ async def retry(self, job: Job, error: str | None) -> None:
         await self._release_job(job.key)
 
         self.retried += 1
-        await self.notify(job)
```

Comment:
why is this removed?

Reply:
The update() function already calls notify().